src/Tasks/InvalidateAllSessionsTask.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace SilverStripe\SessionManager\Tasks;
+
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\Security\RememberLoginHash;
+use SilverStripe\SessionManager\Models\LoginSession;
+
+class InvalidateAllSessionsTask extends BuildTask
+{
+    /**
+     * @var string
+     */
+    private static $segment = 'InvalidateAllSessions';
+
+    /**
+     * @var string
+     */
+    protected $title = 'Invalidate All Login Sessions Task';
+
+    /**
+     * @var string
+     */
+    protected $description = 'Removes all login sessions and “remember me” hashes (including yours) from the database';
+
+    /**
+     * @param HTTPRequest $request
+     */
+    public function run($request)
```

Comment:
```suggestion
    public function run(HTTPRequest $request)
```

Reply:
> Method 'SilverStripe\SessionManager\Tasks\InvalidateAllSessionsTask::run()' is not compatible with method 'SilverStripe\Dev\BuildTask::run()'

We can't add typehints here until they're also in framework.